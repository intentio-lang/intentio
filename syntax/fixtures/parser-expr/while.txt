while { a; b; c; } {
    x
}
