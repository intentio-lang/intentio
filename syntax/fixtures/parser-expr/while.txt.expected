{
  "contents": {
    "_whileBody": [
      {
        "contents": {
          "contents": "x",
          "tag": "Id"
        },
        "tag": "IdExpr"
      }
    ],
    "_whileCondition": {
      "contents": [
        {
          "contents": {
            "contents": "a",
            "tag": "Id"
          },
          "tag": "IdExpr"
        },
        {
          "contents": {
            "contents": "b",
            "tag": "Id"
          },
          "tag": "IdExpr"
        },
        {
          "contents": {
            "contents": "c",
            "tag": "Id"
          },
          "tag": "IdExpr"
        }
      ],
      "tag": "BlockExpr"
    },
    "tag": "WhileExpr"
  },
  "tag": "ExprStmt"
}
